libs/libcoap/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libcoap
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/obgm/libcoap
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)
+PKG_SOURCE_VERSION:=683337890e6fbab1eaa0ef0f2038194fb62b7fa2
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_VERSION).tar.bz2
+
+PKG_MAINTAINER:=Anton Glukhov <anton.a.glukhov@gmail.com>
+PKG_LICENSE:=GPL-2.0+
+PKG_LICENSE_FILE:=LICENSE.GPL
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libcoap
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=CoAP(RFC 7252) library
+  URL:=http://coap.technology/
+endef
+
+define Package/libcoap/description
+  Constrained Application Protocol(RFC 7252) library
+endef
+
+TARGET_CFLAGS += $(FPIC)
+
+CONFIGURE_VARS += \
+	A2X="/usr/bin/a2x"
```

Comment:
getting rid of man page generation would get my vote.  Ideally there's just a different make target in the upstream package that allows this, but when we're never goign to install docs anyway, hacking around tools to generate them is just unnecessary work, and results in building/compiling things that will never be used.  More time, more maintennance.